scripts/generate-paas-manifest.py

```diff
@@ -19,7 +19,9 @@
               help="Output file, if empty the template content is printed to the stdout")
 @click.option('--vars-file', '-f', multiple=True, type=click.Path(exists=True),
               help="Load YAML or JSON variable file")
-def paas_manifest(environment, app, vars_file, out_file):
+@click.option('--var', '-v', multiple=True, type=str,
```

Comment:
`-v` to me is `--verbose` but hey whatever

Reply:
I'm happy to change it to something else

Reply:
To be honest I think I'd prefer grabbing each var explicitly so we know what's expected/allowed. 

Not for this PR but: why are we using `click.option` for this script? We use docopt in our other scripts, we might as well be consistent and use it here too? Or indeed use `click.option` everywhere (docopt is not exactly my favourite tool!).

Reply:
I'm not sure what you mean by explicitly here?

Reply:
`click` is nice but I don't think it's doing much heavy lifting here, looking at it I think we could probably implement the same thing using `argparse` and save ourselves a dependency.

Reply:
I mean having separate options for `--dump-file-name`, `--s3-post-url-data` etc. Long winded and less flexible, but also less risky that the wrong thing will end up in the manifest.

Reply:
Oh that makes sense, I totally get where you're coming from now.

My worry with doing it this way is then we have to specify the arguments in two places; what about instead we looked at raising errors if the manifest template doesn't use one or more of the cars?

Reply:
I think we'd run in to trouble with the other Makefile commands that use this function. Unless we had a separate `s3_app_paas_manifest` function that checked the vars, then called this one? (Ugh!)

The reason I'm being picky about this is that our usage of paas manifests (for which apps, what is expected) is not well documented or tested at the moment (I have that ticket to make example manifests of course!). Perhaps we should just have a comment here describing what's going on and the sort of vars we'd expect.